pkg/components/velero/component_test.go

```diff
@@ -19,7 +19,6 @@ import (
 
 	"github.com/hashicorp/hcl/v2"
 
-	"github.com/kinvolk/lokomotive/pkg/components/internal/testutil"
```

Comment:
Please split moving tests and adding new ones into 2 separate commits, so the addition is clearly visible. Also why do we move it to separate package?